accounts/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
```

Comment:
If this is an admin UI, this is gonna clash with what @naijauser started preparing. Nnamdi, please make sure that you don't both do the same thing.

Reply:
I believe this should still be fine. From what I saw, @naijauser was building the hospital admin dashboard, which I think we could still use. This administrator tool gives us access to all the hospitals, its users, patients, doctors, etc., so I imagine it being be a tool for admins/developers of the Infomed site (us).